index.js

```diff
@@ -93,17 +99,15 @@ class Modal extends Component {
             toValue: toValue ? 1 : 0.8,
             tension: animationTension
           }
-        ).start();
+        ).start(() => {
+          if (toValue) {
+            this.props.modalDidOpen();
+          } else {
+            this.setState({open: false, children: undefined});
+            this.props.modalDidClose();
+          }
+        });
```

Comment:
thanks for the PR! this is definitely a better solution, but could you define a new method on the class instead of copying the callback that is passed to `start()`?